_unittest/test_12_PostProcessing.py

```diff
@@ -498,7 +506,7 @@ def test_09f_update_traces_in_report(self):
     @pytest.mark.skipif(
         config["desktopVersion"] < "2022.2", reason="Not working in non-graphical mode in version earlier than 2022.2."
     )
-    def test_09d_create_monitor(self):  # pragma: no cover
+    def test_09h_create_monitor(self):  # pragma: no cover
```

Comment:
I think we can remove `# pragma: no cover`. To be double checked.
```suggestion
    def test_09h_create_monitor(self):
```